caddyhttp/proxy/reverseproxy.go

```diff
@@ -254,6 +254,10 @@ func (rp *ReverseProxy) copyResponse(dst io.Writer, src io.Reader) {
 
 func copyHeader(dst, src http.Header) {
 	for k, vv := range src {
+		// some predefined headers (e.g. MIME plugin), skip.
+		if _, ok := dst[k]; ok {
+			continue
+		}
```

Comment:
@tw4452852 So you're saying if they're using the `mime` directive, for instance, it might break the logic here even if we ignore / don't modify these headers?